### ISSUE TITLE
Uncaught case

### ISSUE THREAD

**Issue by joseph-allen** [sandsturm/whatsapp-converter#2]
My date '2/21/19' does not get caught by the line
`elif (match.group(3) == '/' and match.group(8) == '- '):`

I get the error:
ValueError: time data '2/21/19' does not match format '%d.%m.%y'

changing to:
`elif (match.group(3) == '/' and match.group(8) == ' -'):`
fixes this for me, note the space before the hyphen

**Comment by sandsturm** [comment]
Hi @joseph-allen, I will take a look at it! Can you post some rows of your original dataset?

**Comment by joseph-allen** [comment]
```
2/20/19, 10:14 PM - Messages to this group are now secured with end-to-end encryption. Tap for more info.
2/20/19, 10:14 PM - You created group "Example chat"
2/21/19, 9:09 AM - Joe: Wow what an amazing chat!
2/21/19, 9:10 AM - Dom: I agree, it is amazing!
2/21/19, 9:12 AM - Joe: You're the best!
2/21/19, 9:12 AM - Dom: No, you are the best!
```

These are from a UK device.

**Comment by sandsturm** [comment]
I am sorry it took so long. I corrected it and uploaded version 0.3.2 of the package to PyPI. I would be happy if you could at least check it, even though you may have found another solution.

**Comment by joseph-allen** [comment]
Nice one, no worries :)
